<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Sign_up</class>
 <widget class="QWidget" name="Sign_up">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>340</width>
    <height>290</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OurChat</string>
  </property>
  <widget class="QPushButton" name="Button_finishsignup">
   <property name="geometry">
    <rect>
     <x>125</x>
     <y>230</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>点击完成注册</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="user_id">
   <property name="geometry">
    <rect>
     <x>95</x>
     <y>140</y>
     <width>150</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="placeholderText">
    <string>帐号</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="user_password">
   <property name="geometry">
    <rect>
     <x>95</x>
     <y>190</y>
     <width>150</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="placeholderText">
    <string>密码</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="name">
   <property name="geometry">
    <rect>
     <x>95</x>
     <y>90</y>
     <width>150</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="placeholderText">
    <string>昵称(请使用英文)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>304</x>
     <y>0</y>
     <width>38</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style:none;
background:transparent;</string>
   </property>
   <property name="text">
    <string>×</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
